backend/src/handlers/user_handler.go

```diff
@@ -17,14 +18,16 @@ func (srv *Server) registerUserRoutes() []routeDef {
 	return []routeDef{
 		{"GET /api/users", srv.handleIndexUsers, true, axx},
 		{"GET /api/users/{id}", srv.handleShowUser, false, axx},
+		{"GET /api/users/{id}/program-history", srv.handleGetUserProgramHistory, false, axx},
```

Comment:
I wonder if `/programs` and `program-history` can do the same thing? Do they need to be separate?

Reply:
After looking at it further I still think we can get what we need from `/programs`. Happy to talk through my throught process.